src/components/checkbox-group/checkbox/checkbox.js

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import Component from './checkbox-component';
-import actions from 'redux_modules/metadata';
+import metadataActions from 'redux_modules/metadata';
+import * as googleAnalyticsActions from 'actions/google-analytics-actions';
```

Comment:
If we are only using `openLayerInfoModalAnalyticsEvent` wouldn't it be better to:
`import { openLayerInfoModalAnalyticsEvent } from 'actions/google-analytics-actions';`
 and
 `const actions = { ...metadataActions, openLayerInfoModalAnalyticsEvent };`?